.tools_versions.yaml

```diff
@@ -1,10 +1,20 @@
+# renovate: datasource=github-releases depName=kubernetes/code-generator
```

Comment:
We still do not have renovate installed in this repo but it's good that you're adding this. 👍